insta_reels_publishing_api_sample/index.js

```diff
@@ -102,8 +132,8 @@ app.post("/uploadReels", async function (req, res) {
         const igResponse = await axios.get(getInstagramAccountUri);
         const hasIgBusinessAccount = igResponse.data.instagram_business_account ? true : false;
         const igUserId = igResponse.data.instagram_business_account.id;
-        const { videoUrl, caption, coverUrl, thumbOffset } = req.body;
-        const uploadParamsString = `caption=${caption}&cover_url=${coverUrl}&thumb_offset=${thumbOffset}&access_token=${req.session.userToken}`;
+        const { videoUrl, caption, coverUrl, thumbOffset, locationId } = req.body;
```

Comment:
I'm making locationId as a variable and setting it to empty string in case it isn't selected. Changes are [here](https://github.com/fbsamples/reels_publishing_apis/pull/31/files#diff-37aac94429bbc3ecc0538e2358d59e9fd77d51b2ae1bdc07aa6314e320288467R134)